### DISCUSSION TITLE
How can I make a suggestion for a feature I coded?

### DISCUSSION THREAD

**Discussion by TKDKid1000** [EssentialsX/Essentials#4006] [Q&A] [answered]
So I coded a little thing that is really handy in essentials. Different join/leave messages for different groups. This wasn’t suggested in the PR suggestions area though, so I’m wondering where exactly I should send this through, and what the PR template is. Any help?

**Comment by pop4959** [comment, marked as answer]
I see you've already made a PR, and that's great if you already have code you wish to have reviewed for addition to Essentials. It's generally also a good idea to make an issue about it beforehand to discuss potential options or receive feedback on whether it's suitable in the first place.
